village/src/components/Smurfs.js

```diff
@@ -1,16 +1,27 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
 
+  state = {
```

Comment:
Instead of keeping separate state here in `Smurfs.js` _and_ `SmurfForm.js`, what if we kept _one_ state in `App.js` and passed state down as props to its child components? Would that solve the issue of our smurfs not rendering when we add one? 😊